pkg/api/featuremgmt_test.go

```diff
@@ -52,44 +52,41 @@ func TestGetFeatureToggles(t *testing.T) {
 	t.Run("toggles hidden by config are not present in the response", func(t *testing.T) {
 		features := []*featuremgmt.FeatureFlag{
 			{
-				Name:    "toggle1",
-				Enabled: true,
-				Stage:   featuremgmt.FeatureStageGeneralAvailability,
+				Name:  "toggle1",
+				Stage: featuremgmt.FeatureStageGeneralAvailability,
 			}, {
-				Name:    "toggle2",
-				Enabled: false,
```

Comment:
having the state inline for tests is nice, but i don't think worth adding the confusion of an enabled flag that is not the source of truth